library/src/commonMain/kotlin/de.westnordost.osmfeatures/Locale.kt

```diff
@@ -0,0 +1,17 @@
+package de.westnordost.osmfeatures
+
+import kotlin.jvm.JvmOverloads
+
+data class Locale @JvmOverloads constructor(
+    val language: String,
+    val region: String? = null,
+    val script: String? = null
+) {
+        companion object {
+            val default: Locale? = null
```

Comment:
I'll look into it

Reply:
See point 4 of my review, I already implemented it if we use String instead of an own Locale data class. And I tend towards that solution, because it is most portable and also forward compatible (e.g. in case iD preset localizations will appear for other weird language/sublanguage combinations that the IETF language tag allows

Reply:
I did merge that now into this branch.